Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/authentication/login/login.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { InputFieldComponent } from '../../shared/input-field/input-field.component';
+import { ButtonComponent } from '../../shared/button/button.component';
+import {
+  FormBuilder,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { AuthenticationService } from '../shared/authentication.service';
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [InputFieldComponent, ButtonComponent, ReactiveFormsModule],
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.css',
+})
+export class LoginComponent implements OnInit {
+  loginForm: FormGroup = new FormGroup({});
```

Comment:
no need to assigne it first to the empty FormGroup, you can set it to the this.fb.group({...}) right here and it will work)